packages/ruleset/src/functions/property-inconsistent-name-and-type.js

```diff
@@ -1,4 +1,4 @@
-const { validateSubschemas } = require('../utils');
+const { getSchemaType, validateSubschemas, SchemaType } = require('../utils');
```

Comment:
needed to update this rule's implementation so that it could retrieve a schema's type even if the type was specified within an allOf list element schema.